apps/wallet/src/ui/app/components/nft-display/index.tsx

```diff
@@ -1,37 +1,60 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
-import cl from 'classnames';
 
-import ExplorerLink from '_components/explorer-link';
-import { ExplorerLinkType } from '_components/explorer-link/ExplorerLinkType';
-import Icon, { SuiIcons } from '_components/icon';
+import { cva } from 'class-variance-authority';
+
+import { NftImage, type NftImageProps } from './NftImage';
 import { useMiddleEllipsis, useNFTBasicData } from '_hooks';
 
 import type { SuiObject as SuiObjectType } from '@mysten/sui.js';
-
-import st from './NFTDisplay.module.scss';
+import type { VariantProps } from 'class-variance-authority';
 
 const OBJ_TYPE_MAX_LENGTH = 20;
 const OBJ_TYPE_MAX_PREFIX_LENGTH = 3;
 
-export type NFTsProps = {
+const containerStyles = cva('flex flex-nowrap items-center', {
```

Comment:
Not blocking, but these components feel like they're not quite broken down into base UI components, so using things like cva here feel a little weird.